core/modules/comment/views/views_handler_argument_comment_user_uid.inc

```diff
@@ -10,6 +10,7 @@
  *
  * @ingroup views_argument_handlers
  */
+#[\AllowDynamicProperties]
 class views_handler_argument_comment_user_uid extends views_handler_argument {
```

Comment:
Wouldn't it make more sense to set `#[\AllowDynamicProperties]` only once for the base class views_handler_argument? If I got things right, this is inherited.